webapp/channels/src/components/preparing_workspace/invite_members.tsx

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useMemo, useEffect} from 'react';
+import {CSSTransition} from 'react-transition-group';
+import {FormattedMessage} from 'react-intl';
+
+import {Animations, mapAnimationReasonToClass, Form, PreparingWorkspacePageProps} from './steps';
+
+import Title from './title';
+import Description from './description';
+import PageBody from './page_body';
+import SingleColumnLayout from './single_column_layout';
+
+import InviteMembersLink from './invite_members_link';
+import PageLine from './page_line';
+import './invite_members.scss';
+
+type Props = PreparingWorkspacePageProps & {
+    disableEdits: boolean;
+    className?: string;
+    teamInviteId?: string;
+    formUrl: Form['url'];
+    configSiteUrl?: string;
+    browserSiteUrl: string;
+}
+
+const InviteMembers = (props: Props) => {
+    let className = 'InviteMembers-body';
+    if (props.className) {
+        className += ' ' + props.className;
+    }
+
+    useEffect(props.onPageView, []);
+
+    const inviteURL = useMemo(() => {
+        let urlBase = '';
+        if (props.configSiteUrl && !props.configSiteUrl.includes('localhost')) {
+            urlBase = props.configSiteUrl;
+        } else if (props.formUrl && !props.formUrl.includes('localhost')) {
+            urlBase = props.formUrl;
+        } else {
+            urlBase = props.browserSiteUrl;
+        }
+        return `${urlBase}/signup_user_complete/?id=${props.teamInviteId}`;
```

Comment:
What is the difference between `browserSiteUrl` and `props.configSiteUrl` ?

It is most likely the SiteUrl has not been set yet so not sure how effective this invite screen is so early in the process

Reply:
yeah, probably no need to use the configsiteurl.

Reply:
Yes, and what is `browserSiteUrl`? Same as `formUrl`? Just confused why we have 3